app/models/AuthorityResource.java

```diff
@@ -187,6 +187,33 @@ public String gndRelationEdges() {
 		return Json.toJson(result).toString();
 	}
 
+	public Integer getBirthYear() {
+		String[] parts = fieldValues("dateOfBirth-dateOfDeath", json)
+			.collect(Collectors.joining())
+			.split("–");
+		return Integer.parseInt(parts[0]);
+	}
+
+	public Integer getDeathYear() {
+		String[] parts = fieldValues("dateOfBirth-dateOfDeath", json)
+			.collect(Collectors.joining())
+			.split("–");
+		if (parts.length > 1 && parts[1].length() == 4) {
+			try {
+				return Integer.parseInt(parts[1]);
+			} catch (NumberFormatException e) {
+				return null;
+			}
+		}
+		return null;
```

Comment:
Like above for `getBirthYear()`:

```suggestion
		return fieldValues("dateOfDeath", json).findFirst()
				.map(AuthorityResource::year)
				.map(Integer::parseInt)
				.orElse(null);
```

At this point, we could extract these two to a single method, passing the field name. There, we could also handle the NumberFormatException. To test these edge cases (missings fields, NumberFormatException, etc.), we should probably add some unit tests in `test/models/ModelTest.java`.